mobile/src/src/components/common/CommentBar/CommentBar.jsx

```diff
@@ -1,6 +1,7 @@
-import React, { useState } from "react";
-import { StyleSheet, View, TextInput } from "react-native";
+import React, { useState, useContext } from "react";
+import { StyleSheet, View, TextInput, Text } from "react-native";
 import CommentBarButtons from "./CommentBarButtons";
+import NavContext from "../../../contexts/NavContext";
 
 /*
 To add the comment bar to your components, you need to pass in the callback function
```

Comment:
Maybe briefly mention sendComment here? I mostly added this paragraph to explain which props are callbacks, cause js doesn't have much clarity about data types.